service/src/integrationTest/java/uk/gov/hmcts/reform/fpl/controllers/ManageLegalCounselControllerTest.java

```diff
@@ -0,0 +1,106 @@
+package uk.gov.hmcts.reform.fpl.controllers;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.springframework.boot.test.autoconfigure.OverrideAutoConfiguration;
+import org.springframework.boot.test.autoconfigure.web.servlet.WebMvcTest;
+import org.springframework.boot.test.mock.mockito.MockBean;
+import uk.gov.hmcts.reform.ccd.client.model.AboutToStartOrSubmitCallbackResponse;
+import uk.gov.hmcts.reform.ccd.client.model.CaseDetails;
+import uk.gov.hmcts.reform.ccd.model.Organisation;
+import uk.gov.hmcts.reform.fpl.model.CaseData;
+import uk.gov.hmcts.reform.fpl.model.LegalCounsellor;
+import uk.gov.hmcts.reform.fpl.model.common.Element;
+import uk.gov.hmcts.reform.fpl.service.legalcounsel.ManageLegalCounselService;
+
+import java.util.List;
+
+import static java.util.Arrays.asList;
+import static java.util.Collections.emptyList;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+import static uk.gov.hmcts.reform.fpl.utils.CoreCaseDataStoreLoader.populatedCaseDetails;
+import static uk.gov.hmcts.reform.fpl.utils.ElementUtils.element;
+
+@WebMvcTest(ManageOrdersController.class)
+@OverrideAutoConfiguration(enabled = true)
+class ManageLegalCounselControllerTest extends AbstractCallbackTest {
```

Comment:
could this have mid event mentioned in the name

Reply:
The thing is I'm also testing "aboutToStart" and "aboutToSubmit" here.

Reply:
Ahh missed that, want to split them out into individual files for each endpoint as we do elsewhere or do you think that it is overkill

Reply:
Let's see how it's going to look like after all the other changes.

Reply:
I think I'll keep it as it is. I think it might be a bit of unnecessary duplication to split these ones.